mopidy_musicbox_webclient/static/js/controls.js

```diff
@@ -383,44 +383,27 @@ function doSingle() {
  * Use a timer to prevent looping of commands
  *********************/
 
-function doSeekPos(value) {
-    var val = $("#trackslider").val();
-    newposition = Math.round(val);
-    if (!initgui) {
-        pausePosTimer();
-        //set timer to not trigger it too much
-        clearTimeout(seekTimer);
-        $("#songelapsed").html(timeFromSeconds(val / 1000));
-        seekTimer = setTimeout(triggerPos, 500);
-    }
-}
-
-function triggerPos() {
-    if (mopidy) {
-        posChanging = true;
-        //        mopidy.playback.pause();
-        //    console.log(newposition);
-        mopidy.playback.seek(newposition);
-        //        mopidy.playback.resume();
-        resumePosTimer();
-        posChanging = false;
-    }
-}
-
-function setPosition(pos) {
-    if (posChanging) {
-        return;
-    }
+function timerCallback(position, duration) {
+    $("#songelapsed").html(timeFromSeconds(position / 1000));
```

Comment:
Since I am not familiar with flow of text update I need some time to get things done.